src/main/kotlin/no/nav/persondataapi/rest/oppslag/OppslagResponseDto.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.persondataapi.rest.oppslag
+
+data class OppslagResponseDto<T>(
+    val error: String? = null,
+    val data: T? = null,
+)
```

Comment:
Åh nei, generics 😬

Jeg ville gjerne at alle oppslagsresponsene skal følge samme form, så dette var min beste effort for å standardisere det på tvers.